diff --git a/hyperparams/ppo.yml b/hyperparams/ppo.yml
index c85a982..2572c4a 100644
--- a/hyperparams/ppo.yml
+++ b/hyperparams/ppo.yml
@@ -383,3 +383,14 @@ CarRacing-v0:
   policy_kwargs: "dict(log_std_init=-2,
                        ortho_init=False,
                        )"
+
+Foos-v0:
+  normalize: true
+  n_envs: 16
+  n_timesteps: !!float 1e6
+  policy: 'MlpPolicy'
+  n_steps: 512
+  gae_lambda: 0.98
+  gamma: 0.99
+  n_epochs: 4
+  ent_coef: 0.0
\ No newline at end of file
diff --git a/utils/hyperparams_opt.py b/utils/hyperparams_opt.py
index bfcec6b..7479f0f 100644
--- a/utils/hyperparams_opt.py
+++ b/utils/hyperparams_opt.py
@@ -32,9 +32,9 @@ def sample_ppo_params(trial: optuna.Trial) -> Dict[str, Any]:
     vf_coef = trial.suggest_uniform("vf_coef", 0, 1)
     net_arch = trial.suggest_categorical("net_arch", ["small", "medium"])
     # Uncomment for gSDE (continuous actions)
-    # log_std_init = trial.suggest_uniform("log_std_init", -4, 1)
+    log_std_init = trial.suggest_uniform("log_std_init", -4, 1)
     # Uncomment for gSDE (continuous action)
-    # sde_sample_freq = trial.suggest_categorical("sde_sample_freq", [-1, 8, 16, 32, 64, 128, 256])
+    sde_sample_freq = trial.suggest_categorical("sde_sample_freq", [-1, 8, 16, 32, 64, 128, 256])
     # Orthogonal initialization
     ortho_init = False
     # ortho_init = trial.suggest_categorical('ortho_init', [False, True])
@@ -68,9 +68,9 @@ def sample_ppo_params(trial: optuna.Trial) -> Dict[str, Any]:
         "gae_lambda": gae_lambda,
         "max_grad_norm": max_grad_norm,
         "vf_coef": vf_coef,
-        # "sde_sample_freq": sde_sample_freq,
+        "sde_sample_freq": sde_sample_freq,
         "policy_kwargs": dict(
-            # log_std_init=log_std_init,
+            log_std_init=log_std_init,
             net_arch=net_arch,
             activation_fn=activation_fn,
             ortho_init=ortho_init,
diff --git a/utils/import_envs.py b/utils/import_envs.py
index faf26f8..1d9dc6b 100644
--- a/utils/import_envs.py
+++ b/utils/import_envs.py
@@ -27,3 +27,8 @@ try:
     import gym_donkeycar  # pytype: disable=import-error
 except ImportError:
     gym_donkeycar = None
+
+try:
+    import gym_foos
+except ImportError:
+    gym_foos = None
\ No newline at end of file
